db/schema.rb

```diff
@@ -10,7 +10,28 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_09_09_083144) do
+ActiveRecord::Schema.define(version: 2020_09_18_092018) do
```

Comment:
why is this being pushed

Reply:
It is already in .gitignore but still it appears.